connect/src/spirc.rs

```diff
@@ -1300,12 +1414,21 @@ impl SpircTask {
         // but to replicate the behaviour of the Android client we have to
         // ignore false values.
         let state = frame.get_state();
-        if state.get_repeat() {
+        let repeat = state.get_repeat();
+        let shuffle = state.get_shuffle();
+
+        if repeat {
             self.state.set_repeat(true);
         }
-        if state.get_shuffle() {
+
+        if shuffle {
             self.state.set_shuffle(true);
         }
+
+        if let Some(callback) = &mut self.session_event_callback {
```

Comment:
Our default state is false, false. We need a way to update the event receivers what our actual state is from a fresh load otherwise they'll have no idea.

Reply:
Ah this is the initialization, OK.